frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -441,3 +441,41 @@ action.GitMachete.CheckoutPreviousAction.undefined.branch-name=No branches befor
 action.GitMachete.CheckoutParentAction.text=Git Machete: Checkout Parent Branch
 action.GitMachete.CheckoutParentAction.description=Checkout parent branch (''{0}'')
 action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') does not have a parent in the branch layout
+
+# Traverse
+
+action.GitMachete.TraverseRepositoryAction.description.text=Starting To Traverse
+action.GitMachete.TraverseRepositoryAction.description.empty-layout=Cannot traverse an empty branch layout
+action.GitMachete.TraverseRepositoryAction.GitMacheteToolbar.text=Traverse
+action.GitMachete.TraverseRepositoryAction.invoked-action.id=GitMachete.{0}
+
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.title=Traverse
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.text.HTML=<html>Do you want to traverse the branches of this layout?</html>
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.ok-text=traverse
```

Comment:
the question is yes or no - I would rather make it `Yes` here (probably there is even no need to specify it - it can be a default value for the dialog)

Reply:
Since subconsciously ppl have a tendency to press Enter/yes, I wanted to provide an extra level of caution by changing the text to `traverse` so that the user would not get surprised by the result.

Reply:
a bit of overengineering to me but if you want it so then let's go for `Yes, traverse`